src/EditorFeatures/CSharpTest/Recommendations/ThisKeywordRecommenderTests.cs

```diff
@@ -526,5 +526,109 @@ void M()
     {
         var c = new C { x = 2, y = 3, $$");
         }
+
+        [WorkItem(725)]
+        [WorkItem(1107414)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public void InExpressionBodiedMembersProperty()
+        {
+            VerifyKeyword(@"
+class C
+{
+    int x;
+    int M => $$");
+        }
+
+        [WorkItem(725)]
+        [WorkItem(1107414)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public void InExpressionBodiedMembersProperty_1()
+        {
+            VerifyKeyword(@"
+class C
+{
+    int x;
+    int M => $$
```

Comment:
This test looks like it's the same as the previous one but with text after the caret. Can you check and see if the keyword recommender tests automatically try the test with the fill file and at the end of the file? If so, you can get rid of the above test.